test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeLogs.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common
+{
+    using System;
+    using System.Linq;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public static class EdgeLogs
+    {
+        public static async Task<IEnumerable<string>> CollectAsync(DateTime testStartTime, string filePrefix, CancellationToken token)
+        {
+            var paths = new List<string>();
+
+            // Save module logs
+            string[] output = await Process.RunAsync("iotedge", "list", token);
+            string[] modules = output.Select(ln => ln.Split(null as char[], StringSplitOptions.RemoveEmptyEntries).First()).Skip(1).ToArray();
+
+            foreach (string name in modules)
+            {
+                string moduleLog = $"{filePrefix}-{name}.log";
+                output = await Process.RunAsync("iotedge", $"logs {name}", token);
```

Comment:
Is this the best way to get the logs? Just wondering..

Reply:
Frankly I'm not sure if there's a better way. I wanted to avoid calling docker commands (IotEdgeQuickstart also avoids docker commands), so this seemed to be the obvious choice. But I'm open to doing it another way.